packages/alcotest-async/alcotest-async.0.8.2/descr

```diff
@@ -0,0 +1,61 @@
+Logo](https://raw.githubusercontent.com/mirage/alcotest/master/alcotest-logo.png)
```

Comment:
I don't think this line should be in the `descr`

Reply:
thanks, fixed